docker-app/qfieldcloud/urls.py

```diff
@@ -29,6 +29,7 @@
 )
 from qfieldcloud.authentication import views as auth_views
 from qfieldcloud.core.views import files_views
+
```

Comment:
Can you please remove this line from the diff.

Reply:
Done!